meta/autoid/autoid_test.go

```diff
@@ -120,3 +122,69 @@ func (*testSuite) TestT(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(id, Equals, int64(6544))
 }
+
+func (*testSuite) TestAsyn(c *C) {
+	driver := localstore.Driver{Driver: goleveldb.MemoryDriver{}}
+	store, err := driver.Open("memory")
+	c.Assert(err, IsNil)
+	defer store.Close()
```

Comment:
`defer store.Close()` under the operation of opening  is clearer. And it's in test that doesn't need to consider performance.